sample/signalr-server-selfhost/Connections/DemoHub.cs

```diff
@@ -0,0 +1,248 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web;
+using Microsoft.AspNet.SignalR;
+using Microsoft.AspNet.SignalR.Hubs;
+
+namespace SignalRServer
+{
+    [HubName("demo")]
```

Comment:
Are all these functions being used in tests? If not please leave only what's being used. If you plan to use some of them later stash the changes and add them as you need them.